controller/monitor/disk_utils.go

```diff
@@ -224,3 +235,14 @@ func getSpdkReplicaInstanceNames(client *DiskServiceClient, diskType, diskName,
 
 	return instanceNames, nil
 }
+
+func generateUniqueFirstFourCharUUID(allDiskUUIDFirstFourCharSet map[string]bool) (string, error) {
+	for i := 0; i < 20; i++ {
+		uuid := util.UUID()
+		prefix := uuid[:4]
```

Comment:
Checked the `[google/uuid](https://github.com/google/uuid)`, `uuid.New()` is based UUID v4. The chance of a prefix collision between two consecutive UUIDs is very low, so I'm okay with relying on retries.